test/test_test.go

```diff
@@ -1,5 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright 2017 Authors of Cilium
+// Copyright 2017-2021 Authors of Cilium
+
+// +build integration_tests
```

Comment:
It'd be nice if we could figure out a way to allow both `go test ./...` and also `ginkgo -focus ...` without adding this extra `-tags integration_tests` boilerplate argument for every time a developer wants to run ginkgo :thinking: 

At the same time, I personally have macros wrapped around the ginkgo invocations anyway so it doesn't directly affect me. And I'd probably rather that unit-testing is the easiest type of testing in order to encourage developers to write more of them.

Reply:
> It'd be nice if we could figure out a way to allow both `go test ./...` and also `ginkgo -focus ...` without adding this extra `-tags integration_tests` boilerplate argument for every time a developer wants to run ginkgo 🤔

Agree, I'd prefer this as well, but I can't find a mechanism to do it, short of providing our own `ginkgo` binary.